### ISSUE TITLE
[Feature]: Validation VS. test metrics implementation

### ISSUE THREAD

**Issue by virginiafdez** [londonaicentre/FLIP#50]
### Description

Currently, there are two curves: a training loss curve, and a validation curve. The validation curve is collecting the metrics from the cros_site_model_eval .py file, which gives you ONE metric at the end of the global training for each trust. This isn't intuitively conveyed in the UI. Validation throughout training should be supported, and then the end TEST results should be reflected in a different way. 

### Acceptance criteria

- [x] For each global round, in addition to the training loss point,  a validation loss point shows up.
- [x] At the end of the total training, the final testing (result of cross_site_model_eval) should be reflected somewhere else. 

### Technical considerations

This issue involves training to the following components:

1. nshflame-base-application: at the end of the execute() function in trainer.py, two metrics should be obtained instead of one:  a loss one, and a validation loss / metric one. Then in execute(), the self.flip.send_metrics should be called TWICE, once with TRAIN_LOSS and one with VAL_METRIC / VAL_LOSS
2. nhsflame-api: the two calls below should be handled separately. Currently, there is a table called "fl_metrics", which saves the loss val for each model, trust and training round. Another table, should be created for fl_val_metrics, with the exact same functions associated to it. 
3. nhsflame-ui: currently, the results from the final evaluation are conveyed via a dot on a curve that has, on the x-axis, all the global rounds. This doesn't make sense, because this evaluation only happens once at the end of the training. This curve should be, instead, populated with the values from the "fl_val_metrics" table above (see 2). We should devise a different way of reflecting the test results: maybe a small cell, with something like FINAL TEST METRIC: trust-1: X, trust-2: Y, or a bar-chart. 

### Assign the relevant label(s) and, if known, Milestone to this feature request

- [x] I confirm I have assigned the relevant label(s) and, if known, Milestone to this feature request

**Comment by virginiafdez** [comment]
This is already happening.
